core/src/main/scala/org/apache/spark/internal/config/Network.scala

```diff
@@ -51,6 +51,21 @@ private[spark] object Network {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString(STORAGE_BLOCKMANAGER_TIMEOUTINTERVAL.defaultValueString)
 
+  private[spark] val NETWORK_EXECUTOR_TIMEOUT =
+    ConfigBuilder("spark.network.executorTimeout")
+      .version("3.4.0")
+      .timeConf(TimeUnit.SECONDS)
+      .createOptional
```

Comment:
QQ: Any particular reason for this to be `s` here and `ms` for `HEARTBEAT_WAITINGLIST_TIMEOUT` ?
Keep both consistent as `ms` ?

Reply:
Update to MILLISECONDS.